user_guide_src/source/outgoing/views.rst

```diff
@@ -19,7 +19,7 @@ Using the example controller you created in the controller page, let’s add a v
 Creating a View
 ===============
 
-Using your text editor, create a file called ``BlogView.php`` and put this in it::
+Using your text editor, create a file called ``blogview.php`` and put this in it::
```

Comment:
I think this should be lower snake case?

Reply:
Personally I like snake case.

Reply:
Fiixed.